AmazonChimeSDKDemo/AmazonChimeSDKDemo/LiveTranscriptionOptionsViewController.swift

```diff
@@ -0,0 +1,202 @@
+        regionPickerView.delegate = self
+        enginePickerView.dataSource = self
+        languagePickerView.dataSource = self
+        regionPickerView.dataSource = self
+        
+        enginePickerView.tag = 1
+        languagePickerView.tag = 2
+        regionPickerView.tag = 3
+    }
+    
+    @IBAction func startTranscriptionButton(_ sender: UIButton) {
+        var url = AppConfiguration.url
+        url = url.hasSuffix("/") ? url : "\(url)/"
+        let encodedURL = HttpUtils.encodeStrForURL(
+            str: "\(url)start_transcription?title=\(meetingId)&language=\(languageSelected)&region=\(regionSelected)&engine=\(engineSelected)")
+        HttpUtils.postRequest(url: encodedURL, jsonData: nil) {_, error in
+            DispatchQueue.main.async {
+                if error == nil {
+                    MeetingModule.shared().dismissTranscription(self)
+                } else {
+                    self.notify(msg: "Transcription call failed, please try again")
+                }
+            }
+        }
+    }
+    
+    override  func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+    }
+}
+
+extension LiveTranscriptionOptionsViewController: UIPickerViewDataSource, UIPickerViewDelegate {
+    func numberOfComponents(in pickerView: UIPickerView) -> Int {
+        return 1
+    }
+    
+    func pickerView(_ pickerView: UIPickerView, numberOfRowsInComponent component: Int) -> Int {
+        switch pickerView.tag {
+        case 1:
+            return engines.count
+        case 2:
+            return languages.count
+        case 3:
+            return regions.count
+        default:
+            return 1
+        }
+    }
+    
+    func pickerView(_ pickerView: UIPickerView, titleForRow row: Int, forComponent component: Int) -> String? {
+        switch pickerView.tag {
+        case 1:
+            return enginesDict[engines[row]]
+        case 2:
+            return languagesDict[languages[row]]
+        case 3:
+            return regionsDict[regions[row]]
+        default:
+            return "Data not found"
+        }
+    }
+    
+    func pickerView(_ pickerView: UIPickerView, didSelectRow row: Int, inComponent component: Int) {
+        let engineSelectedBefore = engineSelected
+        switch pickerView.tag {
+        case 1:
+            engineTextField.text = enginesDict[engines[row]]
+            engineSelected = engines[row]
+            engineTextField.resignFirstResponder()
+        case 2:
+            languageTextField.text = languagesDict[languages[row]]
+            languageSelected = languages[row]
+            languageTextField.resignFirstResponder()
+        case 3:
+            regionTextField.text = regionsDict[regions[row]]
+            regionSelected = regions[row]
+            regionTextField.resignFirstResponder()
+        default:
+            return
+        }
+        
+        if engineTextField.text == "Amazon Transcribe Medical" && engineSelectedBefore == "transcribe" {
```

Comment:
nit: `textField.text` is meant for display purpose, we can check `engineSelected` variable instead.